contrib/ci/fwupd_setup_helpers.py

```
@@ -75,10 +75,12 @@ def detect_profile():
 def pip_install_package(debug, name):
     import subprocess
 
+    env = os.environ.copy()
+    env["PIP_BREAK_SYSTEM_PACKAGES"] = "1"
     cmd = ["python3", "-m", "pip", "install", "--upgrade", name]
     if debug:
         print(cmd)
-    rc = subprocess.call(cmd)
+    rc = subprocess.call(cmd, env=env)
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)